Makefile

```diff
@@ -53,6 +53,11 @@ TIMESTAMP:=$${__TIMESTAMP:-$(shell /bin/date -u "+%Y%m%d%H%MZ")}
 SHORT_SHA=`git rev-parse --short HEAD`
 BRANCH=`git rev-parse --abbrev-ref HEAD | tr -- '-/' '_'`
 
+# In case the top commit Depends-On some framework patch that is not merged yet - override framework master with the
+# proper version
+REQ_LEAPP_PR:=$(shell git show HEAD | grep Depends-On | grep -Eo '[[:digit:]]*')
+REQ_LEAPP_PR_IS_OPEN:=$(shell curl "https://api.github.com/repos/oamg/leapp/pulls/${REQ_LEAPP_PR}" | python -c "import json,sys; print (json.load(sys.stdin)['closed_at'] is None)")
```

Comment:
```suggestion
REQ_LEAPP_PR=$(shell git show HEAD | grep Depends-On | grep -Eo '[[:digit:]]*')
REQ_LEAPP_PR_IS_OPEN=$(shell curl "https://api.github.com/repos/oamg/leapp/pulls/${REQ_LEAPP_PR}" | python -c "import json,sys; print (json.load(sys.stdin)['closed_at'] is None)")
```

Reply:
As I said, I have tried with both Makefile var flavors - I am getting the failure in travis (apparently REQ_LEAPP_PR is empty and REQ_LEAPP_PR_IS_OPEN is badly processed because of that) which I can't reproduce locally if I run the build an run commands directly. 
Still trying though